src/components/unstyled/timeline.css

```diff
@@ -31,5 +31,5 @@
   @apply col-start-2 row-start-2;
 }
 .timeline-end {
-  @apply col-start-1 col-end-4 row-start-3 row-end-4 m-1 self-start justify-self-center;
+  @apply col-start-1 col-end-4 row-start-1 row-end-2 m-1 self-start justify-self-center;
```

Comment:
Why?
This is breaking `timeline-end`
Before:
<img width="925" alt="Screenshot 2024-12-21 at 11 13 15 PM" src="https://github.com/user-attachments/assets/9baf9053-41b4-4b08-8073-51dd218fa877" />

After:
<img width="916" alt="Screenshot 2024-12-21 at 11 13 22 PM" src="https://github.com/user-attachments/assets/333cc214-ccff-4866-ad0a-e702b42547de" />

Reply:
Sorry for the confusion, this change was made only for testing and wasn’t meant to be sent. Thanks for pointing it out!

Reply:
Can you explain what's the issue exactly? I can help.